### DISCUSSION TITLE
how can i see these images

### DISCUSSION THREAD

**Discussion by krishno16** [Cyfrin/foundry-full-course-cu#2364] [General]
![Screenshot 2024-09-02 133706](https://github.com/user-attachments/assets/faa43720-9aca-4fc4-9f56-2c71d83e8004)

**Comment by EngrPips** [comment]
Hello @krishno16, I believe this stuff is in the pipeline to be fixed so I will say patiently wait for this to be fixed and you can see the images or maybe report in the `issue` tab to create a reminder.
